src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private static final int MAX_RADIUS = 20;
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    Circle(String color) {
+        super(color);
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getClass().getSimpleName().toLowerCase()
+                + ", area: " + getArea() + ", radius: " + radius + ", color: " + getColor() + ".");
+    }
+
+    public Circle getRandomCircle() {
```

Comment:
methods that return random figures should be not in models but FigureSupplier class, the same for other figures